crates/macro/src/html_tree/html_node.rs

```diff
@@ -46,12 +69,8 @@ impl ToTokens for HtmlNode {
 impl ToTokens for Node {
     fn to_tokens(&self, tokens: &mut TokenStream) {
         let node_token = match &self {
-            Node::Literal(lit) => quote! {
-                ::yew::virtual_dom::VNode::from(#lit)
-            },
-            Node::Raw(stream) => quote_spanned! {stream.span()=>
-                ::yew::virtual_dom::VNode::from({#stream})
-            },
+            Node::Literal(lit) => quote! {#lit},
+            Node::Raw(stream) => quote_spanned! {stream.span()=>(|| {#stream})()},
```

Comment:
For the same reason as the comment above. Because the braces get stripped first, the statements end up free flow stmt. Have to wrap them otherwise code becomes unreachable.

Reply:
Let's keep it this way for now. I will rework this further in subsequent PRs.

Reply:
NVM. Figured out how to get rid of the closure.